src/components/featured-item/featured-item.scss

```diff
@@ -492,3 +493,17 @@ $featured-item: null !default;
     max-width: 210px;
   }
 }
+
+/* stylelint-disable-next-line selector-not-notation */
```

Comment:
this is the exclusion you referred to in the comment..? stylelint is complaining because of the two :not, merge them and you will not need to exclude any rule. So, like: .ecl-featured-item:not(.ecl-featured-item--highlight, .ecl-featured-item--simple)